S/SHTOOLS/build_tarballs.jl

```diff
@@ -32,6 +34,11 @@ gfortran -fopenmp -shared -o ${libdir}/libSHTOOLS-mp.${dlext} ${whole_archive} $
 
 platforms = expand_gfortran_versions(supported_platforms())
 
+# OpenBLAS 0.3.29 doesn't support GCC < v11 on powerpc64le:
+# <https://github.com/OpenMathLib/OpenBLAS/issues/5068#issuecomment-2585836284>.
+# This means we can't build it at all for libgfortran 3 and 4.
+filter!(p -> !(arch(p) == "powerpc64le" && libgfortran_version(p) < v"5"), platforms)
+
```

Comment:
```suggestion
```
Since we no longer pin OpenBLAS, I think this is no longer necessary.

Reply:
Removed.